platform/src/main/resources/patterns/README.md

```diff
@@ -0,0 +1,44 @@
+**Grok**
+
+**Grok Basics**
+
+Grok works by combining text patterns into something that matches your logs
+
+The syntax for a grok pattern is %{SYNTAX:SEMANTIC}
+
+SYNTAX : Name of the pattern that will match your text. For example, 3.44 will be matched by the NUMBER pattern and 55.3.244.1 will be matched by the IP pattern
+
+SEMANTIC : the identifier you give to the piece of text being matched. For example, 3.44 could be the duration of an event, so you could call it simply duration. Further, a string 55.3.244.1 might identify the client making a request
+
+For the above example, your grok pattern would look something like this:
+```
+%{NUMBER:duration} %{IP:client}
+```
+For more patterns : https://github.com/thekrakken/java-grok/blob/master/src/main/resources/patterns/patterns
+
+**Example**
+
+Syslog : 
+```
+Sep 17 06:55:14,123 pndademocloud-hadoop-dn-3 CRON[25907]: (CRON) info (No MTA installed, discarding output)
+```
+The pattern for this could be :
+```
+%{SYSLOGTIMESTAMP:timestamp} (?:%{SYSLOGFACILITY} )?%{SYSLOGHOST:logsource} %{GREEDYDATA:message}
+```
+Grok returns the following event
+
+```
+logsource : pndademocloud-hadoop-dn-3
+message   : CRON[25907]: (CRON) info (No MTA installed, discarding output)
+timestamp : Sep 17 06:55:14.123
+```
+
+**Regular Expressions**
+
+Grok sits on top of regular expressions, so any regular expressions are valid in grok as well
+
+**Custom Patterns**
+
+Sometimes Hillview doesn’t have a pattern you need. For this, you have an option.
```

Comment:
In the end you probably want to say that given a log pattern and a set of files, what you will get in Hillview is a table with columns corresponding to the names that you gave to the patterns.